src/app/book/pages/book-detail/book-detail.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { BookModel } from '../../../core/models/book';
+
+@Component({
+  selector: 'app-book-detail',
+  templateUrl: './book-detail.component.html',
+  styleUrls: ['./book-detail.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BookDetailComponent implements OnInit {
+  book: BookModel;
+
+  constructor(
+    private route: ActivatedRoute,
+    public router: Router,
+  ) { }
+
+  ngOnInit() {
+    this.book = this.getBook();
+
```

Comment:
remove this empty line and add method type

Reply:
fixed